frontend/src/components/Room/CreateRoom.js

```diff
@@ -107,13 +114,24 @@ const CreateRoom = () => {
                             rules={[
                                 {
                                     required: true,
-                                    message: "Please enter room name.",
+                                    max: 20,
                                 },
                             ]}
                         >
                             <Input placeholder="Enter room name" />
                         </Form.Item>
 
+                        <Form.Item
+                            name="description"
+                            label="Description :"
+                            id="description"
```

Comment:
why **id** is constant? all rooms have same **id**?

Reply:
> why **id** is constant? all rooms have same **id**?

form yae id lay. room tway ka use tae id mha ma hote tar.